docs/advanced.md

```diff
@@ -27,11 +27,11 @@ is run with `./my-cli.js --x=22`.
 Default commands can also be used as a command alias, like so:
 
 ```js
-const argv = require('yargs/yargs')(process.argv.slice(2))
+const argv = require('yargs')(process.argv.slice(2))
   .command(['serve', '$0'], 'the serve command', () => {}, (argv) => {
     console.log('this command will be run by default')
   })
-  .parse()
+  .parse(process.argv.slice(2))
```

Comment:
I suggest `process.argv.slice(2))` in either factory call, or in parse, but not both.